ui/src/main.tsx

```diff
@@ -166,7 +166,7 @@ if (isOnDevice) {
               children: [
                 {
                   index: true,
-                  element: <SettingsAccessIndexRoute.default />,
+                  element: <SettingsAccessIndexRoute />,
```

Comment:
Fixing the eslint warning about the `export const loader` requires a change here to follow the same pattern as other routes.